addons/main.go

```diff
@@ -146,6 +147,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	kappControllerConfigReconciler := &kappconfig.KappControllerConfigReconciler{
```

Comment:
We can't add this to the main without installing the CRD within addons package and updating BOM. See https://github.com/vmware-tanzu/tkg-devel/issues/234

Meanwhile we should add to addons/suite_test.go to ensure it is being tested

Reply:
Removed from main.go